modules/data.atmosphere/R/met.process.R

```diff
@@ -316,6 +323,10 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
 
   model.id  <- convert.input(input.id,outfolder,formatname,mimetype,site.id=site$id,start_date,end_date,pkg,fcn,
                              username,con=con,hostname=host$name,browndog,write=TRUE,lst=lst,lat=new.site$lat,lon=new.site$lon)
+  }else{
+    model.id = ready.id
+    
+    if("CRUNCEP" %in% met){outfolder <- file.path(dir,paste0(met,"_site_",str_ns))}
```

Comment:
For what it's worth, the `sprintf` form of the `paste` statement on this line would be: `sprintf("%s_site_%s", met, str_ns)`.